compiler/cpp/src/thrift/generate/t_py_generator.cc

```diff
@@ -2908,4 +3021,5 @@ THRIFT_REGISTER_GENERATOR(
     "                     Package prefix for generated files.\n"
     "    old_style:       Deprecated. Generate old-style classes.\n"
     "    enum:            Generates Python's IntEnum, connects thrift to python enums. Python 3.4 and higher.\n"
+    "    type_hints:      Generate type hints in write method, including IntEnum generation.\n"
```

Comment:
I would consider re-wording this slightly.

```
Generate type hints in write method, requires and enables Enum generation.
```

IntEnum generated files are not backwards compatible with users that leverage the key_to_value mappings to perform conversions.

Reply:
I like this better than implicitly setting it to true, updated